Makefile

```
@@ -70,7 +70,19 @@ xcode: ## Generate and open Xcode workspace
-archive: ## Create Xcode archive for App Store
+archive: ## Create Xcode archive of the current workspace (direct/Developer ID flavour)
+	@echo "Creating archive at $(ARCHIVE_PATH)..."
+	xcodebuild -workspace "Just Speak to It.xcworkspace" \
+		-scheme "SpeakApp" \
+		-configuration Release \
+		-archivePath $(ARCHIVE_PATH) \
+		archive
+	@echo "Archive created at $(ARCHIVE_PATH)"
+
+.PHONY: archive-appstore
+archive-appstore: ## Regenerate the App Store flavour and create its Xcode archive
```

checkmake (0.3.2)

[warning] Target body for "archive-appstore" exceeds allowed length of 5 lines (9).

(maxbodylength)